packages/frontend-core/src/constants.js

```diff
@@ -70,6 +70,7 @@ export const Features = {
   ENFORCEABLE_SSO: "enforceableSSO",
   BRANDING: "branding",
   SCIM: "scim",
+  SYNC_WEBHOOKS: "syncWebhooks",
```

Comment:
Given that the only type of webhook that we have is an automation webhook, I'd be in favour of scoping this more narrowly to be `syncAutomations`. I'm also really against the 'webhook' terminology in general around automations, as it's not a webhook at all. Where a webhook is a request sent by budibase to an external system. What we have is a http trigger for an automation. 

If we do introduce the concept of real webhooks in future, e.g. sent upon system events, I wouldn't want this name to be confused there. 


*sorry for saying webhook so many times

Reply:
No that makes a lot of sense, happy to change it - just terminology that kind of stuck in my head. Not wedded to the idea!